pom.xml

```diff
@@ -3,26 +3,98 @@
+                <version.camunda.bpm.extensions>1.2</version.camunda.bpm.extensions>
+                <version.camunda.spin>1.6.7</version.camunda.spin>
+                <version.resteasy.spring.boot>3.0.0.Final</version.resteasy.spring.boot>
+                <version.resteasy.jaxrs>3.0.12.Final</version.resteasy.jaxrs>
+                <version.javax.servlet.api>3.1.0</version.javax.servlet.api>
+                <version.javax.ws.rs.api>2.1.1</version.javax.ws.rs.api>
+                <version.org.webjars.bootstrap>2.3.2</version.org.webjars.bootstrap>
+                <version.org.jboss.spec>1.0.0.Final</version.org.jboss.spec>
+                <version.wildfly>11.0.0.Final</version.wildfly>
+                <version.wildfly.maven.plugin>2.0.0.Final</version.wildfly.maven.plugin>
+                <version.jboss-as-maven-plugin>7.9.Final</version.jboss-as-maven-plugin>
+
+                <version.junit>4.12</version.junit>
+                <version.hamcrest.json>0.2</version.hamcrest.json>
+
         </properties>
 
+        <developers>
+                <developer>
+                        <name>Holger Hagen</name>
+                </developer>
+        </developers>
+
+        <scm>
+                <url>http://github.com/taskana/taskanaadapter/tree/master</url>
+                <connection>scm:git:git://github.com/taskana/taskanaadpter.git</connection>
+                <developerConnection>scm:git:ssh://github.com:taskana/taskanaadapter.git</developerConnection>
+        </scm>
+
+        <licenses>
+                <license>
+                        <name>The Apache License, Version 2.0</name>
+                        <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+                </license>
+        </licenses>
+        <dependencyManagement>
+                <dependencies>
+                        <dependency>
+                                <groupId>org.springframework.boot</groupId>
+                                <artifactId>spring-boot-dependencies</artifactId>
+                                <version>${version.spring.boot}</version>
+                                <type>pom</type>
+                                <scope>import</scope>
+                        </dependency>
+                </dependencies>
+        </dependencyManagement>
+
         <modules>
                 <module>taskana-adapter</module>
                 <module>taskana-adapter-camunda-system-connector</module>
                 <module>taskana-adapter-taskana-connector</module>
                 <module>taskana-adapter-sample</module>
-                <module>taskana-adapter-wildfly-example</module>
-                <module>taskana-adapter-test</module>
                 <module>taskana-adapter-camunda-tasklistener</module>
                 <module>taskana-adapter-tasklistener-example</module>
                 <module>taskana-adapter-camunda-outbox-rest</module>
                 <module>taskana-adapter-outbox-rest-spring-boot-starter</module>
```

Comment:
same as above

Reply:
taskana-adapter-outbox-rest-spring-boot-starter will be renamed to taskana-adapter-camunda-outbox-rest-spring-boot-starter with  https://taskana.atlassian.net/browse/TSK-1055